pkg/cmd/operator/analyze.go

```diff
@@ -107,6 +108,12 @@ func (o *AnalyzeOptions) Complete() error {
 func (o *AnalyzeOptions) Run(streams genericclioptions.IOStreams, cmd *cobra.Command) error {
 	klog.Infof("%s version %s", cmd.Name(), version.Get())
 	cliflag.PrintFlags(cmd.Flags())
+	if len(o.ArchivePath) != 0 {
+		indexers := analyze.IndexersFromArchive(o.ArchivePath)
```

Comment:
overall you should get listers as the end result, think `NewDataSourceFromArchivePath` to align with the other PR

Reply:
done